packages/manager/modules/pci/src/projects/project/instances/instance/backup/backup.html

```diff
@@ -35,5 +35,9 @@
             <span data-translate="pci_projects_project_instances_instance_backup_warning_message"></span>
         </oui-message>
 
+        <oui-message type="warning" data-ng-if="$ctrl.instance.flavor.type === 'ovh.iops'">
```

Comment:
Maybe this could be turned to 
```suggestion
        <oui-message type="warning" data-ng-if="$ctrl.instance.isIops()'">
```